### PR TITLE
515 compat 2, ./proc/ boogaloo

### PR DESCRIPTION
**Pull request by S34NW**
## What Does This PR Do
Catches any instances of cursed `./proc/` and just `/proc/` that were used for some reason in callbacks and INVOKE_ASYNC.
I had not checked for these because I didn't think they existed.
Regex used to find them: `CALLBACK\(.*proc\/` and `INVOKE_ASYNC\(.*proc\/` Instances manually updated.

Also updates the CI check to account for these

### CODE REVIEW

**Comment by AffectedArc07** [comment]
> Instances manually updated.

Thank god for that

File: code/modules/mob/living/carbon/human/death.dm [hunk 1]
```diff
@@ -88,7 +88,7 @@
 		dna.species.handle_death(gibbed, src)
 
 	if(SSticker && SSticker.mode)
```
Comment (AffectedArc07) [inline code/modules/mob/living/carbon/human/death.dm:90]:
```suggestion
	if(SSticker.mode)
```
Please while youre here these checks need to be put down

**Comment by S34NW** [comment]
There are still uses of `/proc/` in the code but these are in `sortTim` calls which feed into a `GLOBAL_PROC_REF` so they are fine

**Comment by SteelSlayer** [comment]
Found a couple more while on your branch:

image

**Review by farie82 — APPROVED** [review]

**Pull request merged**